GitCommands/Git/GitModule.cs

```diff
@@ -835,6 +835,25 @@ private IGitItem GetSubmoduleCommitHash(string filename, string refName)
             return null;
         }
 
+        public (int? first, int? second) GetCommitRangeDiffCount(ObjectId firstId, ObjectId secondId)
+        {
+            var args = new GitArgumentBuilder("rev-list")
+            {
+                $"{firstId}...{secondId}",
+                "--count",
+                "--left-right"
+            };
+            var output = _gitExecutable.GetOutput(args);
+
+            var counts = output.Split('\t');
+            if (counts.Length == 2 && int.TryParse(counts[0], out var first) && int.TryParse(counts[1], out var second))
```

Comment:
This is another reason for adding a generic detection of git error and warning messages to `GetOutput` etc.

Reply:
How would it help?